examples/pennylane_universal_classifier.py

```diff
@@ -0,0 +1,474 @@
+r"""
+.. _universal_classifier:
+
+Universal Quantum Classifier 
+============================
+**Author: Shahnawaz Ahmed (shahnawaz.ahmed95@gmail.com)**
+
+A single-qubit quantum circuit which can implement arbitrary unitary
+operations can be used as a universal classifier much like a single
+hidden-layered Neural Network. As surprising as it sounds, 
+`Pérez-Salinas et al. (2019) <https://arxiv.org/abs/1907.02085>`_
+discuss this with their idea of 'data
+reuploading'. It is possible to load a single qubit with arbitrary
+dimensional data and then use it as a universal classifier.
+
+In this example, we will implement this idea with Pennylane - a
+python based tool for quantum machine learning, automatic
+differentiation, and optimization of hybrid quantum-classical
+computations.
+
+Circles
+~~~~~~~
+
+We consider a simple classification problem and will train a
+single-qubit variational quantum circuit to achieve this goal. The data
+is generated as a set of random points in a plane :math:`(x_1, x_2)` and
+labeled as 1 (blue) or 0 (red) depending on whether they lie inside or
+outside a circle. The goal is to train a quantum circuit to predict the
+label (red or blue) given an input point’s coordinate.
+
+.. figure:: ../../examples/figures/universal_circles.png
+   :scale: 65%
+   :alt: circles
+
+
+Transforming quantum states using unitary operations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A single-qubit quantum state is characterized by a two-dimensional state
```

Comment:
This is nice for beginners, but a bit strange if a user has read through a lot of tutorials already that do much more advanced things with qubits, only to hear now what a qubit is...I'd consider removing, or rephrasing as "As a reminder..." and shorten to one sentence.